tilequeue/worker.py

```diff
@@ -161,8 +161,12 @@ def __call__(self, stop):
                 cut_coords.extend(coord_children_range(coord, nominal_zoom))
             max_zoom = 16 - self.metatile_zoom
 
-            assert coord.zoom <= max_zoom, \
-                "Job coordinates above max zoom are not supported"
+            if coord.zoom > max_zoom:
+                self.logger.log(
+                    logging.WARNING,
+                    'Job coordinates above max zoom are not supported, '
+                    'skipping %r > %d' % (coord, max_zoom))
+                continue
```

Comment:
I think I put it in there to check the assumption that tilequeue only receives valid job coordinates. And it did its job of alerting me when I scrolled back through the messages to find out why the worker had stopped doing any work! Unfortunately, that's not the behaviour we want :wink: